beamer/tests/config/util.py

```diff
@@ -18,6 +24,25 @@ def write_keystore_file(path, private_key, password):
     path.write_text(json.dumps(obj))
 
 
+def read_config_state(rpc_file, artifact):
+    root = pathlib.Path(__file__).parents[3]
```

Comment:
763e9b8f8fbfc17c258b286bc6fc18d96d001874

Reply:
then you wouldnt need the last commit actually, same for true the commit before changing the read command start block

Reply:
I am not sure I understand, could you please explain in more detail what you have in mind here?

Reply:
I was thinking, you could add get_root_repo in https://github.com/beamer-bridge/beamer/commit/763e9b8f8fbfc17c258b286bc6fc18d96d001874 and do the changes in other commits as well, also the small start_block change can be moved to de7f8032c60bd18e20810649f4feb51fc50df241 or 9bf96301a2a1172e25569e54c3ee426789f1101e.

Reply:
> I was thinking, you could add get_root_repo in [763e9b8](https://github.com/beamer-bridge/beamer/commit/763e9b8f8fbfc17c258b286bc6fc18d96d001874) and do the changes in other commits as well

The separation here is intentional -- I didn't want to combine the two unrelated changes (move of code, introduction of a new function) into the same commit. In general, I think that's not a good thing because it makes it harder to review, understand the commit changes and provide good commit messages. I left this part as is.

>  also the small start_block change can be moved to [de7f803](https://github.com/beamer-bridge/beamer/commit/de7f8032c60bd18e20810649f4feb51fc50df241) or [9bf9630](https://github.com/beamer-bridge/beamer/commit/9bf96301a2a1172e25569e54c3ee426789f1101e).

Nicely spotted! I folded the start_block change into 9bf9630 and updated the commit message appropriately.